modules/containers-signature-verify-enable.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-container-signature.adoc
+
+[id="containers-signature-verify-enable_{context}"]
+= Enabling signature verification for Red Hat Container Registries
+Enabling container signature validation requires files that link the registry URLs to the sigstore and then specifies the keys which verify the images.
+
+.Procedure
+. Create the files which link the registry URLs to the sigstore and specifies the key to verify the image.
+.. policy.json
```

Comment:
This seems like an unusual structure for our docs, just listing the file name as a sub-step. Maybe an unordered list would be better, unless you need to create them in a specific order. @bmcelvee WDYT?

Reply:
+1 to removing the sub-steps and using command and outputs here

Reply:
Updated to:

![image](https://user-images.githubusercontent.com/16167833/116445676-22bc8000-a824-11eb-93da-2aeed32300ba.png)

Does this work better?